sigs/idp-builders/charter.md

```diff
@@ -1,42 +1,64 @@
 # SIG IDP Builders
 
-This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
-the Roles and Organization Management outlined in [sig-governance].
+This charter adheres to the conventions described in the [SIG Readme](../README.md).
 
-## Scope
+# Community
 
-SIG IDP Builders is responsible for developing common tooling for creating reference CNOE internal developer platforms.
+### Meetings
 
-### In scope
+[Google Calendar Link](https://calendar.google.com/calendar/u/0/embed?src=064a2adfce866ccb02e61663a09f99147f22f06374e7a8994066bdc81e066986@group.calendar.google.com&ctz=America/Los_Angeles)
 
-#### Code, Binaries and Services
+### Slack
+- [Join CNCF Slack Team](https://communityinviter.com/apps/cloud-native/cncf), if you are already not a member.
+- [cncf #cnoe-interest](https://cloud-native.slack.com/archives/C0452H8NTNU)
+
+# Scope
+
+SIG IDP Builders is responsible for developing common tooling, patterns, and practices for creating reference CNOE internal developer platforms.
+
+## Sub-Project: IDPBuilder
+
+### Code, Binaries and Services
 
 - [idpbuilder project](https://github.com/cnoe-io/idpbuilder)
-- related packaging tooling
-- supporting libraries for idpbuilder
+- supporting libraries for idpbuilder (e.g. [argocd-api](https://github.com/cnoe-io/argocd-api))
 
-#### Cross-cutting and Externally Facing Processes
+### Roles
 
-- creating cnoe packaging standards or formats
+- owners:
+  - Manabu McCloskey (@nabuskey)
```

Comment:
Is Manabu still working on the project and be considered as an owner ?

Reply:
Updated with @punkwalker for now